src/main/java/com/xiaomi/infra/pegasus/client/ClientOptions.java

```diff
@@ -34,7 +34,7 @@
   public static final String DEFAULT_META_SERVERS =
       "127.0.0.1:34601,127.0.0.1:34602,127.0.0.1:34603";
   public static final Duration DEFAULT_OPERATION_TIMEOUT = Duration.ofMillis(1000);
-  public static final int DEFAULT_ASYNC_WORKERS = Runtime.getRuntime().availableProcessors();
+  public static final int DEFAULT_ASYNC_WORKERS = 4;
```

Comment:
should `PEGASUS_ASYNC_WORKERS_DEF` in `src/main/java/com/xiaomi/infra/pegasus/rpc/Cluster.java` changes to 4 synchronously ?

Reply:
Yeah, have changed it